admin/config/locales/en.yml

```diff
@@ -200,7 +200,15 @@ en:
           settings: SKU Pricing Settings
           title: Pricing
           info: Prices Info
-          info_message: For products with existing prices, regular and sale prices will be applied to all prices with a minimum quantity of 1.
+          info_message: >
+            For products with existing prices, regular and sale prices
+            will be applied to all prices with a minimum quantity of 1.
+            Sites will be
+          increase: 'Increase all prices by'
+          decrease: 'Decrease all prices by'
+          set: 'Set prices with a minimum quantity of 1 to'
+          flat: '%{currency}'
```

Comment:
This seems a little odd, why not output the currency?

(edit) maybe for dynamic reasons?

Reply:
yeah i was thinking if someone wanted to output something like "$ (USD)" vs "$ (AUD)" rather than "$", on different locales.